src/sbvr-api/permissions.ts

```diff
@@ -1086,53 +1088,41 @@ const getGuestPermissions = memoize(
 );
 
 const getReqPermissions = (req: PermissionReq, odataBinds: ODataBinds = []) =>
-	Promise.join(
```

Comment:
This is a breaking change because it's possible to provide a `req.apiKey` to internal requests and previously it was not required to have an actor set but now it is

Reply:
You mean it's possible pass a request object that has `apiKey` but didn't go through `checkApiKey`?

Reply:
I found a case in https://github.com/balena-io/open-balena-api/blob/v0.31.0/src/platform/jwt.ts#L171-L174
Do you know about other examples?

Do you think doing an optional actor ID lookup if it's not set on the request object is better or just upgrading open-balena-api?

Reply:
I think doing the optional lookup and then removing that in 11.x would be better

Reply:
Done in 089170b47d766a49048bd36bc69b7f12a3a5470d PTAL